app/utils/draft/index.ts

```diff
@@ -17,6 +22,15 @@ export function errorBadChannel(intl: IntlShape) {
     return alertErrorWithFallback(intl, {}, message);
 }
 
+export function errorBadUser(intl: IntlShape) {
```

Comment:
Good question. The message is always saying the user has been deleted, but the actual usecase is when the user "cannot be found". Be it because it is deleted, or because it was never a valid user. Since it is right now only used on link parsing for the go to command, it is more possible that the user is not even created (therefore the `bad user` name of the function). So should we change the message or should we change the function name?

Thoughts @enahum ?